### DISCUSSION TITLE
How to increase max belly size?

### DISCUSSION THREAD

**Discussion by ghost** [Squidpad/BG3DevouringAndDigesting#28] [Q&A] [answered]
Hi! Just wanted to know how I could maybe change the max belly size in the mod. Belly scaling only goes up to a point, and I was wondering if there was a "max belly size" cap in the code somewhere. Could it be through running main.py with different variables? Thanks!

**Comment by Squidpad** [comment, marked as answer]
I'll give a barebones explanation here; we plan on eventually putting a formal tutorial together, but I've been super busy with work and probably will be til the new year.
- Open `remeshed.blend` in Blender v3.6
- for each body you'd like to re-generate, set the value of "Enabled" to `True`
- Set "BelliesCount" to the number of meshes to generate
- Set "BelliesStep" and "BelliesSliderStep" to the difference between the volume of successive bellies
- Run the script from Blender's python console (you can google how to do this)
- Wait for potentially a good bit, depending on your hardware
- replace the `_merged.xml` and `CharacterCreationSharedVisuals.xml` in the mod with the ones generated by the script, and paste the contents of the `table.lua` generated by the script with the similar looking table in `BellyTable.lua`
- replace the `.GR2` files in `...\Generated\Public\DevouringAndDigesting\Assets\Characters\Humans\` with the ones generated by the script.

I haven't done anything with the models in quite a long time, so this is mostly just from glancing at the script again and from memory, but hopefully I'm not forgetting anything.

**Reply by ghost** [reply]
This worked great, thanks! Only real difference from the instructions was changing the various lsx files for each race (since each has a different lsx) instead of a merged one (since the only "merged" file I could find was in templates for the vore potions). Only problem now is the belly model is slightly detacted from the player model; it has a small gap between the top of the stomach and the player, regardless of the belly size. It seems to work fine on dragonborn models, but not the body type 1 fem model. Haven't tried the others. Maybe I edited one of the lsx files wrong, or maybe misclicked in Blender? If this issue sounds familiar, let me know. Thanks again! :)

**Reply by SSS314268** [reply]
sorry for mu dumbass sir. i don't understand how to do it. can someone make a proper guide to teach my how? ToT i'll love you for the rest of my life ( sorry for my poor Eng skill)
